src/test/java/spoon/test/template/TemplateTest.java

```diff
@@ -727,64 +728,14 @@ public void substituteStringLiteral() throws Exception {
 	public void substituteSubString() throws Exception {
 		//contract: the substitution of substrings works on named elements and references too
 		Launcher spoon = new Launcher();
-		spoon.addTemplateResource(new FileSystemFile("./src/test/java/spoon/test/template/testclasses/SubStringTemplate.java"));
```

Comment:
It was not intentional! I probably based this PR on wrong branch. Old tests must remain. There is no reason to remove them. If you would like to have it fixed soon then fix it please somehow. I have only slow access to github web these days... so I can't do it.